zipline/finance/slippage.py

```diff
@@ -77,7 +78,45 @@ def fill_price_worse_than_limit_price(fill_price, order):
     return False
 
 
-class SlippageModel(with_metaclass(ABCMeta)):
+class SlippageModelMeta(ABCMeta):
```

Comment:
@richafrank Thanks for the feedback, I updated based on your comments. I also decided to call this `SlilppageModelMeta` instead of `AssetSlippageMeta` since we are now using it as the metaclass for the core `SlippageModel` class, but let me know if you have feelings against this.